pages/docs/running/orchestration.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: "ELT & ETL Orchestration"
+date: "2021-11-11"
+pullQuote: "It's easy to integrate Grouparoo into your existing ELT/ELT process"
+---
+
+It's easy to integrate Grouparoo into your existing ELT/ELT process! There are 3 main ways tell Grouparoo when to run:
+
+- [Triggering Schedules via the API](#triggering-schedules-via-the-api)
+- [Triggering Schedules via a Query](#triggering-schedules-via-a-query)
+- [Running Grouparoo directly via an Orchestrator](#running-grouparoo-directly-via-an-orchestrator)
+
+## Triggering Schedules via the API
+
+Using the Grouparoo API, you can choose to run a [Schedule](/docs/getting-started/product-concepts#schedule) whenever it is right for you. A common use-case is to trigger all of your Grouparoo Schedules to look for new & updated Records at the conclusion of a [dbt](https://www.getdbt.com/) transform step via and Orchestrator like [Airflow](https://airflow.apache.org/) or [Dagster](https://dagster.io/).
+
+The `/api/v1/schedules/run` endpoint can be used to trigger a Run for your Schedules. You can run all of your Schedules or provide a list of specific `scheduleIds` to run.
+
+First, create an [API Key](/docs/getting-started/product-concepts#api-key) with the `write` permission on `Sources`, and then you can use the API as documented below:
+
+```bash
+# Set Variables
+GROUPAROO_HOST="http://localhost:3000"
+GROUPAROO_API_KEY="ba05b18d9dae4acbb08799e7ef238f0d"
+
+# Request to run all Schedules now
+curl \
+  -X POST \
+  -d "apiKey=$GROUPAROO_API_KEY" \
+"$GROUPAROO_HOST/api/v1/schedules/run"
+
+# Request to view the available schedules
+curl \
+  -X GET \
+"$GROUPAROO_HOST/api/v1/schedules?apiKey=$GROUPAROO_API_KEY"
+
+# Request to run a specific schedule now
+# The names of your schedules will be of the format (source_name)_schedule
+# Provide IDs as a JSON Array
+curl \
+  -X POST \
+  -d "apiKey=$GROUPAROO_API_KEY" \
+  -d "scheduleIds=[\"users_table_schedule\"]" \
+"$GROUPAROO_HOST/api/v1/schedules/run"
+```
+
+More information about using [Grouparoo's REST API can be found here](/docs/support/rest-api), including how to generate API Keys. You can learn more about the specifics of this (or any) API endpoint that your Grouparoo instance provides by visiting the `/swagger` page on your Grouparoo instance.
+
+## Triggering Schedules via a Query
+
+Coming soon.
+
+## Running Grouparoo directly via an Orchestrator
```

Comment:
Same question here. What is in an orchestrator? Can you describe some examples in the sentences below to clarify what we mean?